src/Cli.Tests/ValidateConfigTests.cs

```diff
@@ -145,6 +145,52 @@ public void TestValidateConfigFailsWithInvalidGraphQLDepthLimit(object? depthLim
         }
     }
 
+    /// <summary>
+    /// This Test is used to verify that the validate command is able to catch when data source field or entities field is missing.
```

Comment:
So is this test for both datasource and entities? If so why have a separate test for datasource the other test?